fixtures/jira.yaml

```diff
@@ -1,7 +1,7 @@
 schema: 1
 snyk: 
     orgID: 456
-    severity: critical #threshold for jira ticket creation
+    severityThreshold: critical #threshold for jira ticket creation
```

Comment:
Same this should not be changed. You can create a new fixtures with a the new field